bundler.d/fog.rb

```diff
@@ -1,4 +1,4 @@
 group :fog do
-  gem 'fog', '1.30.0', :require => false
-  gem 'fog-core', '1.29.0'
+  gem 'fog', '1.32.0', :require => false
+  gem 'fog-core', '1.32.0'
```

Comment:
Nope, entirely my fault I think.  I must have misread either the rubygems.org page or gem-compare output.